### DISCUSSION TITLE
Format a column of values ​​in Dbeaver to be output as a decimal

### DISCUSSION THREAD

**Discussion by HebertAguiar** [dbeaver/dbeaver#17761] [Q&A]
Good Morning.
Could you tell me how to format a column of values ​​in Dbeaver to be output as a decimal?
Today it is 123.12 and I need 123,12
Thank you very much in advance

**Comment by LonwoLonwo** [comment]
Hello @HebertAguiar 

You can do it in the Preferences -> Editors -> Data Editors -> Data Formats

You can change Locale, or disable numeric formatting

![2022-09-28 18_14_31-DBeaver Ultimate 22 2 1 - PIPES](https://user-images.githubusercontent.com/45152336/192817603-76f2ae4f-c6e9-44a4-9fe2-cdeb34fad2c8.png)

**Reply by HebertAguiar** [reply]
Thank you very much
